core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/scheduling/ExpansionGreedyRegionPlanGenerator.scala

```diff
@@ -93,7 +93,7 @@ class ExpansionGreedyRegionPlanGenerator(
     * This function builds a region DAG from scratch. It first adds all the regions into the DAG. Then it starts adding
     * edges on the DAG. To do so, it examines each PhysicalOp and checks its input links. The links will be problematic
     * if they have one of the following two properties:
-    *   1. The link's toOp (this PhysicalOp) has and only has blocking links;
+    *   1. The link's toOp (this PhysicalOp) has and only has dependee links;
     *   2. The link's toOp (this PhysicalOp) has another link that has higher priority to run than this link
```

Comment:
what's the difference between those two cases now?

Reply:
The first case is not needed. There will not be an operator that only has dependee input links. Removed this from the scaladoc.